examples/grayscale_video_example.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JsfeatCpp video grayscale example</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <video id="video" autoplay loop muted playsinline></video>
+    <canvas id="canvas"></canvas>
+
+    <script>
+
+        var canvas = document.getElementById('canvas');
+        var ctx = canvas.getContext('2d');
+
+        window.addEventListener('jsfeatCpp-loaded', function (e) {
+            var jsfeatCpp = Module;
+            var imgproc = new jsfeatCpp.imgproc();
+            const U8_t = Module.Types.U8_t.value;
+            const C1_t = Module.Types.C1_t.value;
+            const COLOR_RGBA2GRAY = Module.Colors.COLOR_RGBA2GRAY.value;
+            if (navigator.mediaDevices && navigator.mediaDevices.getUserMedia) {
+                var hint = {
+                    audio: false,
+                    video: true
+                };
+                if (window.innerWidth < 800) {
+                    var width = (window.innerWidth < window.innerHeight) ? 240 : 360;
+                    var height = (window.innerWidth < window.innerHeight) ? 360 : 240;
+
+                    var aspectRatio = window.innerWidth / window.innerHeight;
+
+                    console.log(width, height);
+
+                    hint = {
+                        audio: false,
+                        video: {
+                            facingMode: 'environment',
+                            width: { min: width, max: width }
+                        },
+                    };
+
+                    console.log(hint);
+                }
+
+                navigator.mediaDevices.getUserMedia(hint).then(function (stream) {
+                    video.srcObject = stream;
+                    video.addEventListener('loadedmetadata', function () {
+                        video.play();
+
+                        console.log('video', video, video.videoWidth, video.videoHeight);
+
+                        var canvasWidth = video.videoWidth;
+                        var canvasHeight = video.videoHeight;
+                        canvas.width = canvasWidth;
+                        canvas.height = canvasHeight;
+                        process();
+                    });
+                });
+                function moveBytes(src, dest, width, height) {
```

Comment:
this function should be slower than:
```
function moveGray(src, dest, width, height) {
    var i;
    const destW = new Uint32Array(dest.buffer);
    const alpha = 0xFF000000;  // alpha is the high byte. Bits 24-31
    for (i = 0; i < width * height; i++) {
        const g = src[i];
        destW[i] = alpha + (g << 16) + (g << 8) + g;
    }    
}
```
see example here https://stackoverflow.com/questions/73191734/efficiently-update-a-canvas-with-rgb-or-grayscale-data-but-not-rgba/73231895#73231895
(that code is usd also in the original jsfeat example) but this last not works in our code. Don't know why...

Reply:
fixed with https://github.com/kalwalt/webarkit-jsfeat-cpp/pull/12/commits/9d7424007fc9b62ae3babe7c5c7a8e7b04d664f8